### PR TITLE
Verifier also needs to be trusted by Endorser and Verifier Owner

### PR DESCRIPTION
**Pull request by William-PanWei**
For the situation that the Lead Attester has an internal Verifier, the Lead Attester may need to be evaluated first before it receives the Endorsements and Appraisal Policy of other Attesters. This is also a security enhancement.

### CODE REVIEW

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -212,6 +212,9 @@ Then the Claims Collector combines the Attestation Results into
 the final Evidence of the whole Composite Attester which is sent off to the remote
 Verifier, which might treat the claims obtained from the local Attestation Results
 as if they were Evidence.
+Before receiving the Endorsements and Appraisal Policies for other Attesters,
+to increase the security, the Lead Attester may first generate Evidence about
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md]: This sentence is confusing.  How exactly does it "increase the security" to do things in that order?   And if the other Attesters push their evidence to the lead attester, then the order can't be guaranteed anyway, right?

Comment (William-PanWei) [reply draft-ietf-rats-architecture.md]: In the normal Attester case, the Endorser and Verifier Owner needs to trust the Verifier before sending Endorsement and Appraisal Policy to the Verifier. So it is the same in the case that the Lead Attester has an internal Verifier. "increase the security" may not be appropriate, what I want to express is to establish the trust.

The Lead Attester collecting Evidence of other Attesters happens inside the Composite Attester, so I think it doesn't matter when to collect the Evidence.

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -327,7 +327,11 @@ same way as any other Verifier), for evaluating these Attesters' trustworthiness
 Then the Lead Attesting Environment combines the Attestation Results into
 the final Evidence of the whole Composite Attester which is sent off to the remote
 Verifier, which might treat the claims obtained from the local Attestation Results
-as if they were Evidence.
+as if they were Evidence. In this situation, the internal Verifier may need to be
```
Comment (mcr) [inline draft-ietf-rats-architecture.md]: change internal -> local.

File: draft-ietf-rats-architecture.md [hunk 3]
```diff
@@ -327,7 +327,11 @@ same way as any other Verifier), for evaluating these Attesters' trustworthiness
...
+trusted by the Endorser and Verifier Owner before getting the Endorsements
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md]: This is confusing.   There are multiple Verifiers (local and remote) and thus there can be two Endorsers, and two Verifier Owners.  So "the Endorser" and "the Verifier Owner" is ambiguous.

**Comment by mcr** [comment]
Closed, replaced with #46 after rebase and edits.

**Pull request closed without merging**